test/correctness/gpu_assertion_in_kernel.cpp

```diff
@@ -32,8 +32,9 @@ int main(int argc, char **argv) {
     Func g;
     g(c, x) = f(c, x) * 8;
 
-    g.gpu_tile(x, 8);
-    f.compute_at(g, Var::gpu_blocks()).gpu_threads(x);
+    Var tx;
+    g.gpu_tile(x, tx, 8);
+    f.compute_at(g, x).gpu_threads(x);
```

Comment:
I think "tx" here and in the other tests refers to the threads, right? I.e. the inner var from the split. I find this confusing, because I've also seen "tx" "ty" refer to tile indices when tiling. I suggest using "xi"/"yi" instead of "tx"/"ty".